{"failedTests":[{"testName":"parallel/test-async-context-frame","timestamp":1729236336603,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2019-COMPILED_BY-vs2022 #30896","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2019-COMPILED_BY-vs2022/30896/","buildNumber":30896,"builtOn":"test-rackspace-win2019_vs2019-x64-2","nodeVersion":"24.0.0","commitHash":"d7a0b56b2c7ba02aefef578f3845d29fd33d52b7","callStack":[{"upstreamBuild":66597,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":75290,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":3569,"upstreamProject":"node-daily-master","upstreamUrl":"job/node-daily-master/"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":425,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"flaky","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (1423.1122ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (1686.6442ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (2595.5284ms)\n  ✖ async-hooks\\test-async-local-storage-dgram.js (1460.7532ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-dgram.js failed with exit code 3221225477\n    \n    3221225477 !== 0\n    \n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:935:9)\n        at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (1581.3823ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (1464.605ms)\n  ✖ async-hooks\\test-async-local-storage-errors.js (1293.3188ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-errors.js failed with exit code 3221225477\n    \n    3221225477 !== 0\n    \n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:935:9)\n        at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-gcable.js (1264.0079ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (1178.9736ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (1438.1901ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (1217.2984ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (1158.0378ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (1624.4198ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (1079.6276ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (1149.8658ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (1063.3172ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (1319.4306ms)\n  ✔ parallel\\test-async-local-storage-bind.js (1125.8602ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (1058.97ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (1070.7203ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (1100.2759ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (1358.6427ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (1044.8467ms)\n✖ AsyncContextFrame (30773.9427ms)\nℹ tests 23\nℹ suites 1\nℹ pass 21\nℹ fail 2\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 30787.732\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:49:5\n✖ async-hooks\\test-async-local-storage-dgram.js (1460.7532ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-dgram.js failed with exit code 3221225477\n  \n  3221225477 !== 0\n  \n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:935:9)\n      at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }\n\ntest at test\\parallel\\test-async-context-frame.mjs:49:5\n✖ async-hooks\\test-async-local-storage-errors.js (1293.3188ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-errors.js failed with exit code 3221225477\n  \n  3221225477 !== 0\n  \n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:59:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:935:9)\n      at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":31219.164,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}}]}